crates/nargo/src/cli/verify_cmd.rs

```diff
@@ -112,3 +83,38 @@ pub fn add_dummy_setpub_arr(abi: &mut Abi) {
     // ensuring there will be no conflicts
     abi.parameters.push((RESERVED_PUBLIC_ARR.into(), dummy_arr));
 }
+
+pub fn verify_hlp(
+    prg_dir: &Path, //&std::path::PathBuf,
+    proof_path: std::path::PathBuf,
```

Comment:
It doesn't seem like proof_path or prg_dir are changed here so maybe we can use `AsRef` something like `fn foo<P: AsRef<Path>>(p: P)` ?

- Nit: Can you rename prg_dir to program_dir